committee-steering/sig-governance.md

```diff
@@ -0,0 +1,465 @@
+  opposition of proposals
+  - Attend SIG meetings - attendance of every other SIG meeting is expected
+  - Be responsive to messages sent to SIG mailing list
+- Onboard new technical leads and SIG members
+  - Delegate responsibility and ensure opportunities for members to grow
+
+#### Relationship
+
+Technical leads should meet or communicate regularly as a group to
+ensure they share the same priorities and direction.  Decisions made
+by technical leads should be consistent across the group.  If
+technical leads are providing inconsistent direction, then they must
+meet frequently and align themselves.
+
+#### Selecting technical leads
+
+SIG ~~Foo~~ maintains a group of 3-7 technical leads.
+
+New technical leads should be nominated either by existing technical
+leads or self nominated by sending an email to the tech lead mailing
+list.  Technical leads should achieve consensus on the proposal
+with a fallback to voting.
+
+Members should have shown sustained superior technical judgement
+and leadership within the SIG before being considered for a
+technical lead position.  They should already be consistently
+fulfilling the responsibilities outlined above for 3-6 months.
+The technical lead title is not an honorific title, but instead
+a functional one critical to the sustained success of the SIG.
+
+Before becoming a technical lead, members may reverse shadow an
+existing technical lead until they are familiar with the
+responsibilities.  Reverse shadowing means that they take
+on the responsibilities of tech lead, but their work is subject
+to review.
+
+When possible, it is preferred that the Technical Lead group represent a
+diverse set of organizations.  This is predicated on interest in
+the role from multiple organizations with qualified candidates.  At
+times the *most* qualified candidates may all be from a single
+organization, however interested candidates that meet the
+minimum qualifications exist.  In this event, the technical leads
+may be temporarily comprised of a single organization, but must mentor
+qualified candidates from other organizations with the intention
+of giving them a technical lead role.
+
+### Expiring technical leads
+
+Tech leads that have not been active in an area for 3 should not be
+considered for any decision making and should consider stepping down.
+
+Tech leads that have not been active in an area for 6 months
+should be automatically removed as tls.
+
+If the number of tech leads falls bellow the minimum, or the majority
+of tech leads deem it necessary, then new tech leads may be seeded
+from one or more of:
+
+- top-level OWNERs approvers (owned by the SIG)
+- a proposal with lazy consensus within the SIG.
+- SIG architecture,
+
+Technical leads that have held the position for a year and step down may
+retain the title "SIG ~~Foo~~ technical lead emeritus".
+
+### Member role
+
+Canonical list of members: kubernetes-sig-cli-members@googlegroups.com
```

Comment:
`kubernetes-sig-~~foo~~-members@googlegroups.com` to stay consistent with the example?

Reply:
I've shown my cards...